### ISSUE TITLE
non symmetricity between `fun` and `function` for tuple forces usage of parens

### ISSUE THREAD

**Issue by smoothdeveloper** [fsharp/fslang-suggestions#1336]
those are allegedly similar semantically:
```fsharp
fun a -> //...
function a -> //...
```

but for tuple, it doesn't work
```fsharp
fun a,b -> // error FS0010: Unexpected symbol ',' in lambda expression. Expected '->' or other token.
function a,b -> //...
```

I wonder if this can be fixed without ambiguity, because it would be an incomplete lambda in current version of the language and it would reestablish fuller symmetry with `function` construct.

**The existing way of approaching this problem in F# is** to use `function` or wrap with parens.

Maybe a code fix for putting the parens, or turning into `function` would make sense if we can't make this suggestion work / it is rejected on some grounds.

## Pros and Cons 

**The advantages of making this adjustment to F# are** semantically more consistent.

**The disadvantages of making this adjustment to F# are** may prevent other constructs in the future.

## Extra information

**Estimated cost (XS, S, M, L, XL, XXL):**  M

**Related suggestions:** (put links to related suggestions here)

## Affidavit

Please tick these items by placing a cross in the box:
* [x] This is not a question (e.g. like one you might ask on [StackOverflow](http://stackoverflow.com)) and I have searched StackOverflow for discussions of this issue
* [x] This is a language change and not purely a tooling change.
* [x] This is not something which has obviously "already been decided" in previous versions of F#. I
* [x] I have [searched both open and closed suggestions on this site](http://github.com/fsharp/fslang-suggestions/issues) and believe this is not a duplicate

Please tick all that apply:
* [x] This is not a breaking change to the F# language design
* [ ] I or my company would be willing to help implement and/or test this

## For Readers

If you would like to see this issue implemented, please click the :+1: emoji on this issue. These counts are used to generally order the suggestions by engagement.

**Comment by Martin521** [comment]
`fun` and `function` are actually different in that `fun` expects curried parameters (like a named function), while `function` expects a (single) expression to perform a match on. So the mentioned behavior makes sense.

**Comment by Martin521** [comment]
Actually, I think the use of `function` should be discouraged in favour of proper matches, except for one-liners.
But for `function` one-liners you get a no-no from Fantomas :-(

**Comment by realparadyne** [comment]
No-no from me on discouraging the use of function.  They are brilliant for multiple step transformations starting with a regular match to transform something and then doing another match on that result without having to think up a name to bind the intermediate result(s) to.

Basically, anywhere that an expression result needs to be the 'function' of some singular input value that can have multiple shapes. Without having to come up with a name and use it twice, redundantly, both times.

**Comment by konst-sh** [comment]
to me also the tuple between `fun` and ` ->` looks unambiguous enough to interpret it as function argument pattern,
but it may cause problems if `fun` will become optional #168

**Comment by LyndonGingerich** [comment]
> those are allegedly similar semantically:
> 
> ```fsharp
> fun a -> //...
> function a -> //...
> ```

Could you explain? To the best of my knowledge, the second line is invalid.
